lib/Config.js

```diff
@@ -280,6 +280,39 @@ function requestsConfigAssert(requestsConfig) {
     }
 }
 
+function bucketNotifAssert(bucketNotifConfig) {
```

Comment:
Placing my comment here so replies are easier but it's more of a general comment.

Using assert in production code isn't wrong, in JS anyway, python would be as asserts are ignored under some build conditions. But my real concern is that this same logic would use JOI in other projects but uses assert here. So not sure where to have this discussion but I think using one form of validation would be ideal. Whether that is JOI/other tools, plain if statements or asserts is whatever to me as long as its, for the most part, the same.

If there are other concerns like this project is too old to use JOI or something then sure but I could not find anything in the ticket or descriptions so just pointing this out.

Reply:
Yeah, that's a valid point. I don't think the project is too old to use JOI, but it just wasn't used when it was first written, and asserts have just been continually added. Converting these configs to use JOI is a side project we can tackle when we refactor Cloudserver for node12 or around then

Reply:
Ok, that is fine. Using uniform practices will make reviewing and contributing a lot easier in the long run.